super_gui/src/components/SmartLockUI.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState, useEffect } from "react";
+import { toast } from "react-toastify";
+import Package from "/assets/icons/bxs-package.svg";
+
+const SmartLockUI = () => {
+  const [packageStatus, setPackageStatus] = useState<boolean>(true);
+
+  // Temporary until endpoints are created.
+  const togglePackageStatus = () => {
+    setPackageStatus((prev) => !prev);
+    toast.success("Package retrieved by tenant.");
+    console.log("Package retrieved by tenant.");
+  };
+
```

Comment:
You will get an object back that looks like this. the number is the apartment number and the array contains the locker object.

` { 
        "1" : [ {LOCKEROBJECT} , {LOCKEROBJECT} ] ,
        "2" : [ {LOCKEROBJECT} ]
 }
`

Reply:
You could use a LockerObject prop  to get the data into this object. And do the get request in the parent component and go over the object and make an object for every locker in the object.